packages/storage/amplify_storage_s3_dart/pubspec.yaml

```diff
@@ -10,12 +10,13 @@ environment:
 
 dependencies:
   amplify_core: ^1.0.0-next.0
+  amplify_db_common_dart: ^0.1.0
   async: ^2.8.2
   aws_common: ^0.2.2
   aws_signature_v4: ^0.2.0
   built_collection: ^5.0.0
   built_value: ">=8.4.0 <8.5.0"
-  drift: ^2.1.0
+  drift: any
```

Comment:
Are we sure we want to do any?  This appears like a new approach that we haven't done before so perhaps we should comment the motivations for doing so at least.

Reply:
I think a comment here is fine- shouldn't be in the pubspec, though IMO. `drift` is listed here since the package directly depends on it. `any` means that you impose no constraints on the version you receive. Since we are also directly depending on amplify_db_common (which also depends on `drift`), we could either keep those version constraints in sync or use `any` to defer constraints elsewhere (in this case to `amplify_db_common` and any other transitive dependencies which depend on `drift`).